makefile

```diff
@@ -78,12 +78,12 @@ clean:
 
 # Invoke QEMU wihtout waiting for GDB
 run: build/final.iso
-	qemu-system-i386 -monitor stdio -cdrom ./build/final.iso -boot a
+	qemu-system-i386 -singlestep -monitor stdio -cdrom ./build/final.iso -boot a
```

Comment:
Remove `-singlestep`

Reply:
Will be removed during conflict resolution, as it does not warrant a separate commit.